jjb/app-functions/app-service-configurable.yaml

```diff
@@ -0,0 +1,29 @@
+---
+
+- project:
+    name: app-service-configurable
+    project-name: app-service-configurable
+    project: app-service-configurable
+    mvn-settings: app-service-configurable-settings
+    pre_build_script: 'make test'
+    build_script: 'make build && make docker'
+    cron: 'H 11 * * *'
```

Comment:
Please remove this. We don't want all these jobs on a cron timer.

Reply:
Will do.  This was copied from device-virtual, so someone should look at why it is there and remove it if shouldn't be there.